index.html

```diff
@@ -562,10 +562,14 @@ <h3>
         </p>
       </section>
     </section>
-    <section>
+    <section class="atrisk">
       <h2>
         Installation Events
       </h2>
+      <p>
+        Installation events and supporting the {{BeforeInstallPrompt}} is
```

Comment:
I'm not sure why "installation events" (which includes appinstalled) is optional. I don't think there's anything particularly controversial about appinstalled is there? We should get that supported everywhere.

I'm happy to mark BIP as optional, since it's effectively optional already, as it's up to the user agent to decide when and if to fire it (so any user agent that doesn't implement it could claim conformance already). Marking it as optional just makes that clear (and also makes it legal for window.BeforeInstallPromptEvent to not exist).

Reply:
Is there any privacy risk with `appinstalled`?

Reply:
We (me, @dominickng and @marcoscaceres ) couldn't think of any. You're going to get a CSS media query telling you that the window is standalone as soon as you open the window (which in Chrome Desktop is immediately upon install) anyway, so this doesn't give you any additional information.

Reply:
I’m good then. Merge away!